pom.xml

```diff
@@ -20,7 +20,7 @@ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.wso2.integration.inbound</groupId>
     <artifactId>mi-inbound-smpp</artifactId>
-    <version>1.0.3-SNAPSHOT</version>
+    <version>1.0.2-SNAPSHOT</version>
```

Comment:
Since we already have [v1.0.2](https://github.com/wso2-extensions/esb-inbound-smpp/tree/v1.0.2), why do we decrease the version?

Reply:
The 1.0.2 version need to be re-released with both jar and zip files. Therefore reverted the change made by the maven release plugin.